addons/test_mail/data/test_mail_data.py

```diff
@@ -433,6 +433,62 @@
 
 --------------A6B5FD5F68F4D73ECD739009--"""
 
+MAIL_EML_ATTACHMENT_BOUNCE_HEADERS="""Date: Tue, 24 Dec 2019 11:32:07 +0100 (CET)
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary=16063919151.b32bE0eD.7
+Content-Transfer-Encoding: 7bit
```

Comment:
You could have the date header aligned next to the rest of the file by striping the first line return like so :

    MAIL_EML_ATTACHMENT_BOUNCE_HEADERS="""\
    Date: Tue, 24 Dec 2019 11:32:07 +0100 (CET)
    MIME-Version: 1.0
    Content-Type: multipart/mixed; boundary=16063919151.b32bE0eD.7
    Content-Transfer-Encoding: 7bit
    ..."""

I know the rest of the file doesn't do it but you'll do an happy coder here :)

Reply:
Two happy coders.